### PR TITLE
#162162968 Implement rate article

### PR DESCRIPTION
**Pull request by katunold**
#### What does this PR do?
- Enables users to rate articles
- Write tests for rate article feature
#### Description of Task to be completed
- Enable authenticated users to rate articles
- Ensure that users do not rate articles they authored
#### How should this be manually tested?
- When you click on an article from the home page, you will be redirected to view that article.
- Click the star icon to successfully rate an article.
#### What are the relevant pivotal tracker stories?
[#162162968](https://www.pivotaltracker.com/story/show/162162968)
#### Screenshots
image
image
image

### CODE REVIEW

**Review by kizzanaome — APPROVED** [review]

File: public/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
     <noscript>You need to enable JavaScript to run this app.</noscript>
     <div id="root"></div>
     <script src="https://widget.cloudinary.com/global/all.js" type="text/javascript"></script>
+    <script src="https://code.jquery.com/jquery-1.11.2.min.js"></script>
```
Comment (arthurarty) [inline public/index.html]: @katunold I noticed on this line you are adding a jquery library. However, in the package.json file jquery 3.3.1 is listed as a dependency. Are there any advantages jquery 1.11.12 brings to the project?   Is there any advantage of importing like this as opposed to installing it as a package?

File: src/redux/thunks/index.js [hunk 2]
```diff
@@ -57,6 +58,7 @@ const postDataThunk = (endpoint, data, actionCreator, method) => (dispatch) => {
   const token = localStorage.getItem('user');
   axiosInstance.defaults.headers.common.Authorization = 'Token '.concat(token);
   return axiosInstance[method](endpoint, data).then((response) => {
+      console.log(response.data);
```
Comment (arthurarty) [inline src/redux/thunks/index.js]: @katunold I noticed a console.log on this line.

**Review by arthurarty — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (katunold) [reply public/index.html]: I was using it before but am not using it anymore, let me just kick it out

File: src/redux/thunks/index.js (hunk 2 above)
Comment (katunold) [reply src/redux/thunks/index.js]: All clear now

**Review by arthurarty — APPROVED** [review]

**Pull request merged**